conn-sdk-cli/readmegen/generate.go

```diff
@@ -86,10 +86,45 @@ func Generate(opts GenerateOptions) error {
 }
 
 var funcMap = template.FuncMap{
-	"formatCommentYAML": formatCommentYAML,
-	"formatValueYAML":   formatValueYAML,
-	"zeroValueForType":  zeroValueForType,
-	"args":              args,
+	"formatCommentYAML":            formatCommentYAML,
+	"formatValueYAML":              formatValueYAML,
+	"zeroValueForType":             zeroValueForType,
+	"args":                         args,
+	"describeParameterRequirement": describeParameterRequirement,
```

Comment:
Could this be something like `isRequired` and the function return a boolean?

Reply:
I'm also for `isRequired`, even if it makes the template a bit more verbose (which, tbh, will make it clearer IMO). Think of readmegen as a tool that can be used in different settings, we already use it to generate documentation on conduit.io for instance. I wouldn't tailor functions to the one template that we are using right now, but rather provide general functions that can be useful in many templates.

Reply:
We can do it that way too.:)